src/main/java/Deadline.java

```diff
@@ -0,0 +1,16 @@
+public class Deadline extends Task {
+    protected String dueDate;
+
+    public Deadline(String description) {
+        super(description.substring(9, description.indexOf("/") - 1));
```

Comment:
Would it be better if you have a fixed FINAL variable for "9", this could come in handy there are changes in the future